plugin/src/software/aws/toolkits/eclipse/amazonq/configuration/customization/CustomizationUtil.java

```diff
@@ -68,9 +69,13 @@ public static void validateCurrentCustomization() {
             Customization currentCustomization = Activator.getPluginStore()
                     .getObject(Constants.CUSTOMIZATION_STORAGE_INTERNAL_KEY, Customization.class);
 
-            for (final Customization validCustomization : customizations) {
-                if (validCustomization.getArn().equals(currentCustomization.getArn())) {
-                    return;
+            if (currentCustomization.getProfile() != null) {
```

Comment:
this would be null though right for a customer because the old arn did not have any profile information to beefin with.